integr8ly/keycloak-operator
# INTLY-971 RHSSO 7.3

INTLY-971 Update to use the RHSSO 7.3.GA release template

We currently use an RHSSO 7.2 release template in the operator,
however there is a new 7.3 release with a template. This change
updates the operator to use the new 7.3 release.

The 7.3 template does not use the apiVersion group
'template.openshift.io', so we have to update the REST service
that creates the processed template to use the /oapi REST
endpoint for processed templates.

The template in this commit is taken from:

https://github.com/jboss-container-images/redhat-sso-7-openshift-image/blob/v7.3.0.GA/templates/sso73-x509-postgresql-persistent.json

Verification:
- Run the operator against a cluster `make setup/dep cluster/prepare code/run`
- Create a Keycloak CR `cluster/create/examples`
- Ensure the deployed SSO instance uses RHSSO 7.3 by checking the
    deployment and the RHSSO console.
